test/index.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+const Controller = require('../');
+const Base = require('hof-controllers').base;
+
+describe('Confirm Controller', () => {
+
+  let controller;
+  let req;
+  let res;
+
+  beforeEach((done) => {
+    req = request();
+    res = response();
+    controller = new Controller({
+      route: '/foo',
+      steps: {
+        '/': {
+
+        },
+        '/one': {
+          fields: ['field-one']
+        },
+        '/two': {
+          fields: ['field-two']
+        },
+        '/done': {  }
+      }
+    });
+    controller._configure(req, res, done);
+  });
+
+  it('extends hof-controllers.base class', () => {
+    expect(controller).to.be.an.instanceOf(Base);
+  });
+
+  describe('locals', () => {
+
+    beforeEach(() => {
+      sinon.stub(Base.prototype, 'locals').returns({ super: true });
```

Comment:
I agree. Mostly because I always pad my curlies (not a euphemism!) and the rule annoys me. 

I've fixed it now anyway.